### DISCUSSION TITLE
Appium 2.0 plugin versioning ?

### DISCUSSION THREAD

**Discussion by RSM-SBA** [appium/appium#16030] [Q&A] [answered]
I want to ask if it will be possible with appium 2.0 to install different version of plugins (i.e. XCUITest driver).
So that i an eventually rollback my appium setup to a working state, if the new driver has a breaking change or a major bug.

**Comment by KazuCocoa** [comment, marked as answer]
You can manage `APPIUM_HOME` environment variable to manage dependencies.
https://www.youtube.com/watch?v=fN658Jgl2u0
Previously that was an argument, but appium 2.0 will eventually handle it as an environment variable.

**Reply by jlipps** [reply]
You can also always install a specific driver version, e.g., `appium driver install xcuitest@2.54.1`

**Reply by RSM-SBA** [reply]
Thanks, that was what i ment :)
